Makefile

```diff
@@ -38,9 +38,9 @@ TIMESTAMP_SERVER = 'http://timestamp.verisign.com/scripts/timstamp.dll'
 #
 # Detect OS
 #
-OS = unkown
+OS = unknown
 OS_UNAME := $(shell uname -s)
-ifneq (,$(findstring CYGWIN_NT,${OS_UNAME}))
+ifneq (,$(filter CYGWIN_NT% MSYS_NT%,${OS_UNAME}))
```

Comment:
I see why this is needed here. I hope mage will help us here also to lessen such stupid pains.

And good catch with the typo. :)